core/vm/state/impl/state_tree_impl.hpp

```diff
@@ -23,7 +23,7 @@ namespace fc::vm::state {
    public:
     explicit StateTreeImpl(const std::shared_ptr<IpfsDatastore> &store);
     StateTreeImpl(const std::shared_ptr<IpfsDatastore> &store, const CID &root);
-    ~StateTreeImpl() = default;
+    ~StateTreeImpl() override = default;
```

Comment:
looks like when base ~ is virtual there is no need in defining default ~ in an inherited class at all

Reply:
removed